wiki/BanchoBot/de.md

```diff
@@ -1,25 +1,158 @@
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+Es gibt insgesamt 7 Stati, die mit dem Befehl `!stats` angezeigt werden können: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying und Playing. Ein Beispiel für das Senden dieses Befehls für einen Benutzer mit einem verfügbaren Status ist unten dargestellt:
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy is Playing:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+### Where
+
+```
+!where <Benutzername>
+```
+
+`!where` zeigt den aktuellen Standort des eingegebenen Benutzers an. Standardmäßig wird hier nur das Land des Benutzers angezeigt. Wenn der Benutzer jedoch die Option `Deinen Standort mit anderen teilen` aktiviert hat, wird auch seine Stadt angezeigt. Ein Beispiel für das Senden dieses Befehls ist unten dargestellt:
+
+```
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
+```
+
+### FAQ
+
+```
+!faq <Eintrag>
+```
+
+```
+!faq list
+```
+
+`!faq` zeigt den Inhalt eines Eintrags an. Alternativ dazu kann das Argument `list` verwendet werden, um alle verfügbaren Einträge anzuzeigen. Standardmäßig antwortet BanchoBot auf Englisch, aber es ist möglich, eine Antwort in einer anderen Sprache zu erhalten, indem man dem Eintrag den [Zwei-Buchstaben-Code](/wiki/Article_styling_criteria/Formatting#locales) der betreffenden Sprache als Präfix voranstellt. Beispiele für das Senden dieses Befehls sind unten aufgeführt:
+
+```
+13:03 pippi: !faq peppy
+13:03 BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+```
+
+```
+13:04 pippi: !faq ru:lines
+13:04 BanchoBot: Умещайте свои мысли в меньшее количество строк, чтобы не получить сайленс.
+```
+
+### Report
+
+*Für Informationen darüber, was es wert ist, zu melden, siehe: [Melden von schlechtem Verhalten](/wiki/Reporting_bad_behaviour).*
+
+```
+!report <Benutzer> <Grund>
+```
+
+`!report` benachrichtigt das [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) über das unangemessene Verhalten eines Benutzers. Wenn ein Benutzer Leerzeichen in seinem Benutzernamen hat, werden diese durch Unterstriche ersetzt (z. B. wird aus `really cool username` `really_cool_username`). Um einen Moderator zu melden, kontaktiere das [Account-Support-Team](/wiki/People/The_Team/Account_support_team#support@ppy.sh). Ein Beispiel für die Meldung eines normalen Benutzers durch BanchoBot ist unten dargestellt:
```

Comment:
```suggestion
`!report` benachrichtigt das [Globale Moderationsteam](/wiki/People/The_Team/Global_Moderation_Team) über das unangemessene Verhalten eines Benutzers. Wenn ein Benutzer Leerzeichen in seinem Benutzernamen hat, werden diese durch Unterstriche ersetzt (z. B. wird aus `wirklich cooler benutzername` `wirklich_cooler_benutzername`). Um einen Moderator zu melden, kontaktiere das [Account-Support-Team](/wiki/People/The_Team/Account_support_team#support@ppy.sh). Ein Beispiel für die Meldung eines normalen Benutzers durch BanchoBot ist unten dargestellt:
```
1. The `GMT` article uses a translated version as the title
2. Not too strong about this suggestion. Either is fine for me.